frontend/integration-tests/protractor.conf.ts

```diff
@@ -69,8 +69,8 @@ const testSuites = {
   olmFull: suite([
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/descriptors.scenario.ts',
     '../packages/operator-lifecycle-manager/integration-tests/scenarios/operator-hub.scenario.ts',
-    '../packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts',
-    '../packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts',
+    // '../packages/operator-lifecycle-manager/integration-tests/scenarios/global-installmode.scenario.ts',
+    // '../packages/operator-lifecycle-manager/integration-tests/scenarios/single-installmode.scenario.ts',
```

Comment:
Not this way, use `xdescribe` in the scenario itself. Then it gets gets status disabled in the report so we know a test is disabled.

Reply:
ok, fixed